addons/mail/static/src/models/rtc.js

```diff
@@ -1293,6 +1296,19 @@ registerModel({
         pingInterval: attr({
             compute: '_computePingInterval',
         }),
+        /**
+         * The protocols for each RTC ICE candidate types.
+         */
+        protocolsByCandidateTypes: attr({
+            default: {
+                'host': "",
```

Comment:
Why don't we display HOST instead? Displaying nothing is confusing, and it makes the code harder with the condition on protocol.